collect_app/src/main/java/org/odk/collect/android/views/ODKView.java

```diff
@@ -483,7 +484,20 @@ public void highlightWidget(FormIndex formIndex) {
                 scrollTo(0, qw.getTop());
 
                 ValueAnimator va = new ValueAnimator();
-                va.setIntValues(getResources().getColor(R.color.red), getDrawingCacheBackgroundColor());
+                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
+                    // only for kitkat and newer versions
+                    va.setIntValues(getResources().getColor(R.color.red), getDrawingCacheBackgroundColor());
+                    } else {
+                    TypedValue a = new TypedValue();
```

Comment:
If I don't have any idea what name I should use I prefer a name that comes from an object, in that case, it would be `typedvalue` or `tv` I think it's better than just `a`.

Reply:
Yes @grzesiek2010, I'll change the name now, I'll follow this in future. Thanks.